operator/internal/manifests/storage/var.go

```diff
@@ -99,6 +101,11 @@ const (
 	// KeySwiftPassword is the secret data key for the OpenStack Swift password.
 	KeySwiftUsername = "username"
 
+	saTokenVolumeName         = "bound-sa-token"
+	saTokenVolumeK8sDirectory = "/var/run/secrets/kubernetes/serviceaccount"
+	saTokenVolumeOcpDirectory = "/var/run/secrets/openshift/serviceaccount"
+	saTokenExpiration         = 3600
```

Comment:
We need to investigate if we can rely on a default constant here. IIRC some provides like GCP allow you to set the expiration date on their side. If these two sides are related maybe we need to give the freedom back to keep in sync. Will lookup for you and inform you on this thread ASAP.

Reply:
I was researching this and I was not able to find anything on this being a configurable field from the provider side 🤔